MyToolsYourToolsBackend/MyToolsYourToolsBackend.API/Controllers/NotificationsController.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+using MyToolsYourToolsBackend.Application.Dtos;
+using MyToolsYourToolsBackend.Application.Services;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace MyToolsYourToolsBackend.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [EnableCors("MyToolsYourTools")]
+    public class NotificationsController : ControllerBase
+    {
+        private INotificationService _notificationService;
+        private IUserService _userService;
+
+        public NotificationsController(INotificationService notificationService, IUserService userService)
+        {
+            _notificationService = notificationService;
+            _userService = userService;
+        }
+       
+
+        [HttpGet("{userId}")]
+        public IActionResult GetUserNotifications(Guid userId)
+        {
+            if(_notificationService.userHasNotifications(userId))
+            {
+            return Ok(_notificationService.GetNotificationsForUser(userId));
+            }
+            else return NotFound("no notifications");
+        }
+
+
+        
+        [HttpPost("add")]
```

Comment:
dobra praktytka jest nie uzywanie czasowników w urlach.
https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9